addons/binding/org.openhab.binding.upb/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Can you include the about.html in the file (a new requirement :-) ) and make sure the file ends with a newline